terms.html

```diff
@@ -50,46 +50,48 @@
 
   <dd>
 <a href="https://en.wikipedia.org/wiki/Public_key_infrastructure">Public key
-infrastructure</a> that does not rely on traditional
-<a href="https://en.wikipedia.org/wiki/Certificate_authority">certificate
-authorities</a> because it uses <a>decentralized identifiers</a> and
-<a>DID documents</a> to discover and verify <a>public key descriptions</a>.
+infrastructure</a> that does not rely on traditional <a
+href="https://en.wikipedia.org/wiki/Certificate_authority">certificate
+authorities</a> because it uses <a>decentralized identifiers</a> and <a>DID
+documents</a> to discover and verify <a>public key descriptions</a>.
   </dd>
 
   <dt><dfn data-lt="did controllers|did controller(s)">DID controller</dfn></dt>
 
   <dd>
-An entity that has the capability to make changes to a <a>DID document</a>.
-A <a>DID</a> might have more than one DID controller. The <a>DID controller(s)</a>
-can be denoted by the optional `controller` property at the top level of the <a>DID
-document</a>. Note that one <a>DID controller</a> might be the <a>DID subject</a>.
+An entity that has the capability to make changes to a <a>DID document</a>. A
+<a>DID</a> might have more than one DID controller. The <a>DID controller(s)</a>
+can be denoted by the optional `controller` property at the top level of the
+<a>DID document</a>. Note that a <a>DID controller</a> might be the <a>DID
```

Comment:
```suggestion
<a>DID</a> might have more than one DID controller. The DID controller(s)
can be denoted by the optional `controller` property at the top level of the
<a>DID document</a>. Note that a DID controller might be the <a>DID
```

remove self-referencing definition